src/Icicle/Repl.hs

```diff
@@ -100,8 +109,8 @@ sourceParse = mapLeft ReplErrorCompile . P.sourceParseQT "repl"
 sourceDesugar :: P.QueryTop' -> Either ReplError P.QueryTop'
 sourceDesugar = mapLeft ReplErrorCompile . P.sourceDesugarQT
 
-sourceReify :: P.QueryTop'T -> Either ReplError P.QueryTop'T
-sourceReify = return . P.sourceReifyQT
+sourceReify :: P.QueryTop'T -> P.QueryTop'T
+sourceReify = P.sourceReifyQT
```

Comment:
I did actually have that, but I liked that none of the reexports for the repl said QT at the end, so left it like that.